lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -72,20 +73,27 @@
                 leaveTeam: function (event) {
                     event.preventDefault();
                     var view = this;
-                    $.ajax({
-                        type: 'DELETE',
-                        url: view.teamMembershipDetailUrl.replace('team_id', view.model.get('id'))
-                    }).done(function (data) {
-                        view.model.fetch()
-                            .done(function() {
-                                view.teamEvents.trigger('teams:update', {
-                                    action: 'leave',
-                                    team: view.model
-                                });
+                    ViewUtils.confirmThenRunOperation(
+                        gettext('Leave this team?'),
+                        gettext('Leaving a team means you can no longer post on this team, and your spot is opened for another learner.'),
```

Comment:
"If you leave, you can no longer post in this team's discussions. Your place will be available to another learner."